.github/workflows/java_tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: Data Plane tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  java_tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build Docker image
```

Comment:
you should first run tests using `mvn clean verify -B`

Reply:
Done in Dockerfile https://github.com/pierDipi/knative-eventing-kafka-broker/pull/1/commits/f5a5b98ec7df9447b249ef96d74fc816acd90b0d

Reply:
Ok so this dockerfile is just for testing purpose then?

Reply:
That Dockerfile is only for GitHub action